build-system/test-configs/forbidden-terms.js

```diff
@@ -1235,8 +1103,6 @@ const forbiddenTermsSrcInclusive = {
       'extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js',
       'extensions/amp-iframe/0.1/amp-iframe.js',
       'extensions/amp-script/0.1/amp-script.js',
-      'extensions/amp-sidebar/0.1/amp-sidebar.js',
-      'extensions/amp-sidebar/0.2/amp-sidebar.js',
```

Comment:
Bravo! I don't think this kind of a cleanup would've been possible (or would've ever happened) earlier.